.github/workflows/CI.yml

```
@@ -98,7 +98,7 @@ jobs:
       - name: "Upload coverage data to Codecov"
         if: matrix.julia-version == '1.10' && matrix.depwarn != 'depwarn=error'
         continue-on-error: true
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
```

```suggestion
        uses: codecov/codecov-action@v4
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
```

Right now this (expectedly?) fails with:
```
Error: Codecov token not found. Please provide Codecov token with -t flag.
```